### ISSUE TITLE
First run after install is slow without any feedback to the user

### ISSUE THREAD

**Issue by bserem** [mmunz/ddev-backstopjs#3]
Right after `ddev get mmunz/ddev-backstopjs` and `ddev restart` the restart takes a long time. I suppose it downloads backstop. There is nothing displayed to the end user however. Is the default ddev output when downloading images supresed somehow?

It's not a big deal, but definitely a good thing to have, especially for people on slow connections

**Comment by mmunz** [comment]
Yes, ddev doesn't seem to print anything while base images are downloaded. I think this might be because the image is built and not just downloaded.
I added a note to README.md telling the user that ddev restart might take some time when the image was not downloaded before.
